github/repos_rules.go

```diff
@@ -118,15 +118,20 @@ func (r *RepositoryRule) UnmarshalJSON(data []byte) error {
 	case "creation", "deletion", "required_linear_history", "required_signatures", "non_fast_forward":
 		r.Parameters = nil
 	case "update":
-		params := UpdateAllowsFetchAndMergeRuleParameters{}
-		if err := json.Unmarshal(*RepositoryRule.Parameters, &params); err != nil {
-			return err
+		if RepositoryRule.Parameters == nil {
+			r.Parameters = nil
+			return nil
+		} else {
```

Comment:
One of the points was to remove the "else" and un-indent the following section, please:

```suggestion
		}
```

Reply:
You’re totally right, I think I had a brain freeze. Don’t know why I kept an else with a return in there.